src/app/lib/config/services/thai.js

```diff
@@ -1,8 +1,11 @@
+import { thai as thaiScript } from '@bbc/gel-foundations/scripts';
```

Comment:
Curious why youve done this here for this one, but not `latin` or `arabic`

Reply:
Because the [`const` below](https://github.com/bbc/simorgh/pull/1584/files#diff-b616aad94f6a2934cb808262980e897dR3) is called `thai` there is a naming conflict. So either we can rename the script to be more explicit EG: `thaiScript` or we could change the const declaration to be `thaiConfig` or `thaiServiceConfig`

Reply:
Ohhh gotcha, fine by me :)